src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { LoginComponent } from './components/LoginComponent';
+import { RegistrationComponent } from './components/RegistrationComponent';
+import { Star } from '../../components/Start';
+
+import styles from './Login.css';
+
+export const Login = () => {
+  const [isLogin, setIsLogin] = React.useState(true);
```

Comment:
`useState` лучше заимпортить выше и использовать без `React.` для единообразия. В инпуте именно так сделано.
```suggestion
  const [isLogin, setIsLogin] = useState(true);
```

Reply:
Да, что-то это я странно написал) поправил